proto/identity/associations/association.proto

```diff
@@ -11,11 +11,25 @@ option java_package = "org.xmtp.proto.identity.associations";
 // The identifier for a member of an XID
 message MemberIdentifier {
   oneof kind {
-    string address = 1;
+    string ethereum_address = 1;
     bytes installation_public_key = 2;
+    Passkey passkey = 3;
   }
 }
 
+// Passkey identifier
+message Passkey {
+    bytes key = 1;
+    optional string relying_partner = 2;
+}
+
+// List of identity kinds
+enum IdentifierKind {
+  IDENTIFIER_KIND_UNSPECIFIED = 0; // Ethereum on old clients
```

Comment:
Q: for old clients, can we check if their identity is eth to set them as `IDENTIFIER_KIND_ETHEREUM`?